client/src/components/new-request/analyze-results.vue

```diff
@@ -179,16 +191,16 @@ export default class AnalyzeResults extends Vue {
     scrollingContainer: false
   }
   contents: string = ''
+  finalName: string = ''
   highlightCheckboxes: boolean = false
   issueIndex: number = 0
-  originalName: string | null = null
+  originalName: string | null = ''
```

Comment:
No need to change this but FYI...

TS already knows this is a string (since you initialize it), and I've seen some linter rules complain about this redundancy, so I wouldn't declare this as a string.

Also, string can be null, so you don't need the `| null`. (In fact, any JS object can be null, and since everything is an object, I think you never need `| null`.)
https://www.w3schools.com/js/js_datatypes.asp#:~:text=In%20JavaScript%20null%20is%20%22nothing,typeof%20null%20is%20an%20object.